docs/management/admin/endpoint-protection-rules.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[endpoint-protection-rules]]
+= Endpoint protection rules
+
+Endpoint protection rules are prebuilt rules designed to help you manage and respond to alerts generated by {elastic-endpoint}, the installed component that performs {elastic-defend}'s threat monitoring and prevention. These rules include the <<endpoint-security, {elastic-defend}>> rule as well as additional detection and prevention rules for different {elastic-defend} protection features.
+
+NOTE: To receive {elastic-endpoint} alerts, you must install the Endpoint agent on your hosts (see <<install-endpoint>>).
```

Comment:
I'd bump the admonition up to `IMPORTANT` since this is a requirement for getting Endpoint alerts. (Same suggestion for [this line](https://github.com/elastic/security-docs/pull/6100/files#diff-2660cdd6a483bf299687cd255fea85908e3e47c4e2aad201cb73f4c193aaec29R6) in the Serverless docs).

```suggestion
IMPORTANT: To receive {elastic-endpoint} alerts, you must install the Endpoint agent on your hosts (see <<install-endpoint>>).
```